common/utils/pom.xml

```diff
@@ -51,6 +51,20 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-text</artifactId>
     </dependency>
+    <dependency>
```

Comment:
How much deps do we add to the client by doing this?

Reply:
~400 Kb from commons-io and ~1Mb from Ivy